torch/utils/bottleneck/__main__.py

```diff
@@ -223,9 +223,10 @@ def main():
     # if their execution times are very different.
     cuda_prof_exec_time = cpu_time_total(autograd_prof_cuda)
     cpu_prof_exec_time = cpu_time_total(autograd_prof_cpu)
-    pct_diff = cuda_prof_exec_time - cpu_prof_exec_time / cuda_prof_exec_time
-    if abs(pct_diff) > 0.05:
-        print_autograd_prof_summary(autograd_prof_cpu, 'CPU', autograd_prof_sortby, autograd_prof_topk)
+    if cuda_prof_exec_time > 1e-6:
```

Comment:
why not

```
if cuda_prof_exec_time > 0:
```

Reply:
I did such a test. In python  `1.0/1e-8 = 100'000'000.0`
But `1.0/1e-9 = 999999999.9999999`. 
Then consider it use second, 1e-6 should be enough, or should it be 1e-8?

Reply:
I'm confused by what your test means. If we're fixing a zero-division error then we should check for zeros